index.js

```diff
@@ -13,7 +13,7 @@ module.exports = (nextConfig = {}) => {
       ];
 
       if (fileLoader) {
-        const path = 'static/svg/';
+        const path = useDefaultPath ? 'static/media/' : 'static/svg/';
```

Comment:
Couple of concerns about adding this config option.

A `useDefaultPath` config option might not necessarily be easily understood by consumers and might be missed for many users who are pinned to versions of next with the caching issue. The best solution is probably one that doesn't require a configuration adjustment to simply get the expecting caching behavior.

A config flag like this doesn't tend to scale well. While it might be improbably the API would need to support something other than `static/svg/` and `static/media/`, if it did it might need to change in an unintuitive way to support it.

There might be a simpler solution that doesn't require any additions to the configuration options. Let's simply try another path that might be supported in the versions (12 - 13) where we're seeing the issue. I believe updating the path to `static/image/` might resolve the issue without any other changes required. Initial tests seem to confirm `cache-control` headers are set with correct values using this path.

Reply:
Is there a specific reason why the default `static/media/` path should not be used?

Reply:
🤔 Maybe I'm missing something here. From the outline of this PR, I assumed the `static/media/` path would not work for < 12.x versions of next and that's why there was an addition made to the config.

Honestly, I'm not overly concerned about what the path is. The goal here should be to fix the caching issue without introducing anything new to the API and publish a patch version. If the path your suggesting does all of that, we're good to go. All we need is exactly what you mentioned above...remove the option and use the default path 👍.

Reply:
I’m sorry if I misled you. I have uploaded the solution, which will not introduce anything new to the API.

Reply:
No worries and nothing to be sorry about at all. Change looks good 👍 . Great catch and appreciate the fix!